app/(root)/layout.tsx

```diff
@@ -1,10 +1,9 @@
-import Navbar from '../../components/shared/navbar/Navbar'
+import Navbar from '@/components/shared/navbar/Navbar'
 import React from 'react'
```

Comment:
configuring absolute imports & module path aliases in project from tsconfig.json
[Ref](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases)